s3.js

```diff
@@ -0,0 +1,38 @@
+require("dotenv").config();
```

Comment:
S3.js could be an helper, create a folder like helper to put all separate helper functions